DataFormats/PatCandidates/interface/TriggerObject.h

```diff
@@ -117,37 +117,40 @@ namespace pat {
     /// Special methods for 'l1extra' particles
 
     /// General getters
-    const reco::CandidateBaseRef& origObjRef() const { return refToOrig_; };
+    const reco::CandidatePtr& origObjRef() const { return refToOrig_; };
```

Comment:
Since this is also unused, maybe it would be good to deprecate as well?

Reply:
How did you determine it was unused?

Reply:
I'm trusting `git cms-checkdeps -a` to checkout any downstream code, and none of it failed to compile. I guess there is a chance `auto` saves the day here, will actually search...

Reply:
Were you talking about removing the method `origObjRef` or the member data `refToOrig_`? `refToOrig_` definitely can't be removed as all the code changes you made use it.

Reply:
the method